karma.conf.js

```diff
@@ -31,7 +31,10 @@ module.exports = function (config) {
       require('@angular-devkit/build-angular/plugins/karma')
     ],
     client:{
-      clearContext: false // leave Jasmine Spec Runner output visible in browser
+      clearContext: false, // leave Jasmine Spec Runner output visible in browser
+      jasmine: {
+        random: false
+      }
```

Comment:
Do you really need to deactivate random order of unit tests?

It can certainly be a pain in the ass during development but unit tests should be independent of each other so randomization shouldn't break anything and might help to discover unintended side effects.

Reply:
to be honest, i cant even remember why I did this in the first place. I changed it back